delivery_readmes/DELIVERY.README.Sarek.txt

```diff
@@ -0,0 +1,113 @@
+
+├──ProjectID
+	├── 00-Reports
+        ├── Sample1
+        ├── Sample1.lst
+        ├── Sample1.md5
+        ├── Sample2
+        ├── Sample2.lst
+        ├── Sample2.md5
+	...
+	...
+        ├── SampleN
+        ├── SampleN.lst
+        └── SampleN.md5
+
+
+The root folder contains following files as well:
+
+├──ProjectID
+        ├── ACKNOWLEDGEMENTS.txt
+        ├── DELIVERY.README.SAREK.txt
+        ├── miscellaneous.lst
+        └── miscellaneous.md5
+
+--ACKNOWLEDGEMENTS.txt
+A file describing what to include in any publications using this data.
+
+--miscellaneous.lst
+A list of the files of the delivery, excluding raw data.
+
+--miscellaneous.md5
+A file to check the integrity of the miscellaneous.lst file.
+
+
+File integrity checks
+---------------------
+It is extremely important to check the integrity of the delivered data using the .md5 files. To do this, run the following:
+md5sum -c file_with_checksums.md5
+For more information, please see https://ngisweden.scilifelab.se/resources/data-delivery/
+
+
+====================================================================
+ProjectID -> 00-Reports
+====================================================================
+00-Reports folder contains the reports with the sample and sequence QC. It also contains a project summary report and MultiQC reports.
+
+├── 00-Reports
+	├── ProjectName_project_summary.html
+	├── ProjectName_sample_info.txt
+	├── ProjectName_lane_info.txt
+	├── ProjectName_library_info.txt
+	├── ProjectName_multiqc_report.html
+	├── ProjectName_SarekGermline_multiqc_report.html
+	└── manifestFiles
+		└── <ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
+
+--ProjectName_project_summary.html
+A report summarizing information about the project
+
+--ProjectName_[sample/lane/library]_info.txt
+Tab-separated text files for each table found in the report, convenient for loading into Excel
+
+--ProjectName_multiqc_report.html
+Report summarizing general quality statistics based on the bioinformatics tools FastQC and FastQ Screen.
+
+--ProjectName_SarekGermline_multiqc_report.html
+Report summarizing bioinformatic analysis results for all samples
+
+--<ProjectID>.02-FASTQ.<flowcell>.<ProjectID>_<BCL-CONVERSION-ID>_<lane-number>_manifest.txt
+A file summarizing the run with information about instruments and library.
+
+====================================================================
+ProjectID -> SampleN
+====================================================================
+Each sample directory contains the following subfolders:
+
+├──SampleN
+	├──01-SarekGermline-Results
+	└──02-FASTQ
+
+--------------------------------------------------------------------
+ProjectID -> SampleN -> 01-SarekGermline-Results
+--------------------------------------------------------------------
+Contains output from the sarek pipeline. Refer to
+https://github.com/nf-core/sarek/blob/2.6.1/docs/output.md for output description.
```

Comment:
Pinned version. See also above about just listing the top-level folder name (I think you can just remove this section in fact and keep this sentence at the end of the section above.

Reply:
Updated!